docs/001_develop/03_client-capabilities/007_forms/002_smart-forms/01_json_schema.mdx

```diff
@@ -0,0 +1,317 @@
+The examples provided are simple, but it is worth exploring these sections in more depth.
+:::
+
+
+## Examples 
+
+### Server-side JSON schema
+
+Here is an example of a simple table defined in an application's **-tables-dictionary.kts** file. It is a simple table with three fields:
+
+**smart-forms-examples-tables-dictionary.kts**
+```kts
+ tables {
+    table(name = "SIMPLE_TRADE", id = 10_000) {
+        field("SIMPLE_TRADE_ID", STRING).sequence("ST")
+        field("QUANTITY", DOUBLE)
+        field("SIDE", STRING)
+        primaryKey("SIMPLE_TRADE_ID")
+    }
+    ...
+}
+```
+
+The data for this table is provided by a query in the application's **-dataserver.kts** file. This is the data source:
+
+**smart-forms-examples-dataserver.kts**
+```kts
+dataServer {
+    query("ALL_SIMPLE_TRADES", SIMPLE_TRADE)
+    ...
+}
+```
+
+The event that enables you to insert the data from the form into the database table is provided in the application's **-eventhandler.kts** file. This is the data source:
+
+**smart-forms-examples-eventhandler.kts**
+```kts
+eventHandler {
+    eventHandler<SimpleTrade>("SIMPLE_TRADE_INSERT", transactional = true) {
+        onCommit { event ->
+            val details = event.details
+            entityDb.insert(details)
+            ack()
+        }
+    }
+    ...
+}
+...
+```
+
+To insert the automatically-generated JSON schema for this resource in your **template** file, use the code below:
+
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration:
+```html
+<foundation-form
+  design-system-prefix="rapid"
+  resourceName="EVENT_TRADE_INSERT"
+></foundation-form>
+```
+
+Usage:
+```typescript
+@customElement({
+  name: 'form-example',
+  template: html`
+    <foundation-form
+      design-system-prefix="rapid"
+      resourceName="EVENT_TRADE_INSERT"
+    ></foundation-form>
+  `,
+})
+export class FormExample extends GenesisElement {}
+```
+
+</TabItem>
+<TabItem value="react">
+
+Declaration:
+```html
+<foundation-form
+  design-system-prefix="rapid"
+  resourceName="EVENT_TRADE_INSERT"
+></foundation-form>
+```
+
+Usage:
+```typescript
+export default function FormExample({}) {
+  return (
+    <foundation-form
+      design-system-prefix="rapid"
+      resourceName="EVENT_TRADE_INSERT"
+    ></foundation-form>
+  );
+}
+```
+
+</TabItem>
+<TabItem value="angular">
+
+Declaration
+```html
+<foundation-form
+  design-system-prefix="rapid"
+  resourceName="EVENT_TRADE_INSERT"
+></foundation-form>
+```
+
+Usage
+```typescript
+import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'my-root',
+  template: `
+    <foundation-form
+      design-system-prefix="rapid"
+      resourceName="EVENT_TRADE_INSERT"
+    ></foundation-form>
+  `,
+  styleUrls: ['./my.component.css'],
```

Comment:
if we don't have the css file can we just remove it?

Reply:
[36a4841](https://github.com/genesiscommunitysuccess/docs/pull/1872/commits/36a4841084273021e30e1d3b10e5dd88d6a271bc)